web/src/main/java/org/cbioportal/web/mixin/SampleListMixin.java

```diff
@@ -0,0 +1,21 @@
+package org.cbioportal.web.mixin;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import org.cbioportal.model.CancerStudy;
+
+public class SampleListMixin {
+
+    @JsonIgnore
+    private Integer listId;
+    @JsonProperty("sampleListId")
+    private String stableId;
+    private String category;
+    @JsonIgnore
+    private Integer cancerStudyId;
+    @JsonProperty("cancerStudyId")
```

Comment:
This will be a little confusing until database schema are updated to reflect api field naming. cancerStudyId refers to the (internal) integer id in the database tables, but cancerStudyIdentifier to the stable name string in the API. Let's not forget to change the database fields too, as discussed during the evaluation of the standard.